sdk/keyvault/azkeys/client.go

```diff
@@ -1138,8 +1138,8 @@ type ReleaseKeyOptions struct {
 	// Version is the version of the key to release
 	Version string
 
-	// Enc is the encryption algorithm used to protected exported key material.
-	Enc *ExportEncryptionAlg `json:"enc,omitempty"`
+	// Algorithm is the encryption algorithm used to protected exported key material.
+	Algorithm *ExportEncryptionAlg `json:"algorithm,omitempty"`
```

Comment:
Is the tag change needed?  How is the value sent over the wire?

Reply:
Convenience models don't go over the wire so, I don't know why we have the tags. I assumed the purpose was to help applications de/serialize.

Reply:
Good point.  We can leave it as-is, but you bring up an interesting point that these tags might be superfluous.